server/routes/index.js

```diff
@@ -69,6 +69,11 @@ router.use('/contests', contestsRouter.removeGroupRouter)
 // router.use('/contests', contestsRouter.getLeaderboardRouter)
 // router.use('/contests', contestsRouter.getQuestionLeaderboardRouter)
 
+router.use('/questions/editor_questions', questionsRouter.getEditorQuestions)
```

Comment:
```suggestion
router.use('/questions', questionsRouter.getEditorQuestions)
```

Reply:
uniform code

Reply:
I had made this for testing.cuz it was clashing with /questions/:question_id route...forgot to change

Reply:
this problem occured before....the solution is:
```
get /questions/xyz
get /questions/abc
get/questions/
```
Always use `getSingleEntity` type of APIs after listing all the same type of clashing requests before that....otherwise server gets confused whether 'abc' or 'xyz' is a id too

Reply:
Yeah I know..i had kept it for testing.. I have already committed